plugins/testdata/relation_table.md

```diff
@@ -9,6 +10,8 @@ This table depends on [`test_table`](test_table.md).
 ## Columns
 | Name          | Type          |
 | ------------- | ------------- |
-|string_col|String|
 |_cq_id (PK)|UUID|
-|_cq_fetch_time|Timestamp|
+|_cq_parent_id|UUID|
+|_cq_source_name|String|
```

Comment:
I guess I'll get to it as I read on, but I'm wondering what `_cq_source_name` contains and why we need it